frontend/components/utils/FormatCurrency.vue

```diff
@@ -34,6 +36,19 @@ export default Vue.extend({
         },
     },
     computed: {
+        formattedNumber(): number {
+            if (!this.value.toFixed()) {
+                return 0;
+            }
```

Comment:
Why would you want to return unformatted `0` from the `formattedNumber`?

Reply:
I made it return `this.value` instead of 0.